src/subsplit_dag.hpp

```diff
@@ -118,6 +184,10 @@ class SubsplitDAG {
   // the SubsplitDAGNode of the corresponding child.
   void IterateOverRootwardEdgesAndParents(const SubsplitDAGNode *node,
                                           const EdgeAndNodeLambda &f) const;
+  // Iterate over the rootward edges, supplying the parent node id, child node id,
```

Comment:
This says "rootward edges" but the method says "LeafwardEdges". Is that in conflict?